src/assets/projectWeekBannerAssets.tsx

```diff
@@ -27,21 +27,25 @@ export const projectWeekPupilText = (
 
 export const projectWeekStudentText = (
   <>
-    Hier finden Kurse im Rahmen des neuen Programms &quot;Lern-Fair Fokus &ndash; die
-    digitalen Fokuswochen&quot; statt, wozu du mehr Informationen auf unserer
-    Website{' '}
+    In den Fokuswochen von 
```

Comment:
it looks a bit "ugly", that the blankspaces are also part of the link, but it is not too bad. But if you want, you can fix it. Otherwise great, thanks a lot.